draft-ietf-dprive-dnsoquic.md

```diff
@@ -247,6 +247,10 @@ another port MUST NOT be port 53. This recommendation against use of port 53
 for DoQ is to avoid confusion between DoQ and the use of DNS over UDP
 {{!RFC1035}}.
 
+In the stub to recursive scenario, the use of port 443 as a mutually agreed 
+alternative port can be operationally beneficial, since port 443 is less likely
+to be blocked than other ports.
+
```

Comment:
Yes. But "DoQ" can be differentiated from "H3" by looking at the ALPN extension in the TLS payload of the Initial packet. That's significantly more involved than looking at a port number, but the QUIC initial encryption key can be derived from the a QUIC-version specified constant and the initial connection ID, and the ALPN is encoded as a clear-text extension in the TLS "Client Hello" message carried in the initial packet. Firewalls could go look into that. Of course, clients and servers could also use the "encrypted client hello" extensions to hide the ALPN, launching the next round of tit-for-tat.